src/main/java/org/example/algorithms/Tarjan's Algorithm/Tarjan's Algorithm.java

```diff
@@ -0,0 +1,65 @@
+class Solution {
```

Comment:
File name should not contain any special characters.

Rename file to _TarjanAlgorithm.java_ and place it in folder _src/main/java/org/example/algorithms/_